classpractice/full-name/src/test/java/org/karolina/FullNameTest.java

```diff
@@ -0,0 +1,141 @@
+package org.karolina;
+
+import org.junit.jupiter.api.function.Executable;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+class FullNameTest {
+
+    @ParameterizedTest
+    @MethodSource ("testCasesWithNullsInNotAllowedPlaces")
+    public void AllArgumentsInOverloadedConstructorsAreNotNull (Executable executable) {
+        var calledConstructor = executable;
+
+        assertThrows(NullPointerException.class, calledConstructor);
```

Comment:
sprawdź też `message`, bo `NullPointerException` potrafi mieć message `null` i przy debugowaniu to jest dramat xD